e2e/rbd.go

```diff
@@ -1984,6 +1986,158 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("create a encrypted block pvc and verify the image size", func() {
+				err := deleteResource(rbdExamplePath + "storageclass.yaml")
+				if err != nil {
+					framework.Failf("failed to delete storageclass: %v", err)
+				}
+				err = createRBDStorageClass(
+					f.ClientSet,
+					f,
+					defaultSCName,
+					nil,
+					map[string]string{"encrypted": "true", "encryptionType": util.EncryptionTypeBlock.String()},
+					deletePolicy)
+				if err != nil {
+					framework.Failf("failed to create storageclass: %v", err)
+				}
+
+				//nolint:goconst // The string "1Gi" is used multiple times in rbd.go, so it's not a const value.
+				pvcSize := "1Gi"
+				imageSize := 1*helpers.GiB + util.GetLuksHeaderSize()
```

Comment:
check if helper is having any menthods to parse string and convert it to in that avoid bugs or else we need to remember that we need to change size in two places.